Gomfile

```diff
@@ -0,0 +1,24 @@
+gom 'github.com/Sirupsen/logrus', :commit => '07d998d174c4e2dc90e2f1989a20724220bca1ff'
+gom 'github.com/aws/aws-sdk-go/aws', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/endpoints', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/protocol/ec2query', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/protocol/query/queryutil', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/protocol/rest', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/signer/v4', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/private/waiter', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/aws/aws-sdk-go/service/ec2', :commit => '328e030f73f66922cb9c1357de794ee1bf0ca2b5'
+gom 'github.com/go-ini/ini', :commit => '2e44421e256d82ebbf3d4d4fcabe8930b905eff3'
+gom 'github.com/hashicorp/errwrap', :commit => '7554cd9344cec97297fa6649b055a8c98c2a1e55'
+gom 'github.com/hashicorp/go-multierror', :commit => 'd30f09973e19c1dfcd120b2d9c4f168e68d6b5d5'
+gom 'github.com/jmespath/go-jmespath', :commit => '3433f3ea46d9f8019119e7dd41274e112a2359a9'
+gom 'gopkg.in/yaml.v2', :commit => '7ad95dd0798a40da1ccdff6dff35fd177b5edf40'
+gom 'gopkg.in/fsnotify.v1'
```

Comment:
this is related to the viper change FYI, just mentioning this since my PR for that will come soon, so thanks adding it  :+1: